.github/workflows/release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Create a release
+
+env:
+  BRANCH: release
+  VERSION: 3.3.1
```

Comment:
How to appropriately regex the BRANCH here?

How to make the VERSION come dynamically?